lib/index.js

```diff
@@ -290,23 +435,74 @@ module.exports = class OtpRunner {
       await this.fail('Build graph failed! Please see logs.')
     }
     this.status.graphBuilt = true
+
+    await this.updateStatus(
+      'Graph built successfully!',
+      this.manifest.runServer ? 70 : 90
+    )
+
+    // add various follow-up tasks that can occur independently of the server
+    // startup
+    this.postBuildTasks.push(this.uploadGraphObj())
+    this.postBuildTasks.push(this.uploadBuildLogs())
+    this.postBuildTasks.push(this.createAndUploadBundle())
+    this.postBuildTasks.push(this.uploadGraphBuildReport())
+  }
+
+  /**
+   * Make OTP server arguments according to the specified OTP version
+   */
+  makeServeArgs () {
+    const args = getBaseOTPArgs(this.manifest.jarFile)
+    if (this.isUsingOtp2x()) {
+      args.push('--load')
+      args.push(this.getTargetFolder())
+    } else {
+      args.push('--server')
+      args.push('--graphs')
+      args.push(this.manifest.baseFolder)
+      args.push('--router')
+      args.push(this.manifest.routerName)
+    }
+    return args
+  }
+
+  /**
+   * Run various tasks in parellel after completing the graph build step of this
```

Comment:
typo: parellel

Reply:
Fixed in 2610ee9df6cb1878f067a5e2c1c277eed33836fe